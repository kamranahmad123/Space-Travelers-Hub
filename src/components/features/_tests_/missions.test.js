import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../../redux/store';
import '@testing-library/jest-dom';
import Missions from '../../rockets/missions';
describe('Missions component', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <Missions />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('that jest is working', () => {
    expect(true).toBe(true);
  });
});
