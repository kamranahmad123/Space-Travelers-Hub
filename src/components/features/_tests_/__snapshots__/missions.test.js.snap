// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dragon component matches snapshot 1`] = `
<div
  className="missions-area"
>
  <table
    className="missions-table"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
          Join/Leave Mission
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
